src/main/java/ch/njol/skript/conditions/CondIsInteractable.java

```diff
@@ -0,0 +1,56 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import org.bukkit.Material;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+
+@Name("IsInteractable")
+@Description("Checks wether or not a block is interactable")
+@Examples({"on block break:",
+			"    if event-block is interactable:",
+  			"        cancel event",
+			"        send \"You cannot break interactable blocks!\""})
+@RequiredPlugins("Paper 1.13+")
+public class CondIsInteractable extends PropertyCondition<ItemType> {
```

Comment:
Docs should include a version as well:
```suggestion
@RequiredPlugins("Paper 1.13+")
@Since("INSERT VERSION")
public class CondIsInteractable extends PropertyCondition<ItemType> {
```

Reply:
What would be added in the @Since(""), should I put the version after the current one?

Reply:
just what I did in the suggest "INSERT VERSION" (we insert the version when we release)

Reply:
Ah okay, will add now.